pages/_app.js

```diff
@@ -112,29 +138,103 @@ export default function App({ Component, pageProps }) {
       isoDate: new Date().toISOString(),
     };
 
-    setEmotionEntries([newEntry, ...emotionEntries]);
+    if (useExampleData) {
+      setEmotionEntries([newEntry, ...emotionEntries]);
+    } else {
+      const response = await fetch("/api/emotionEntries", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(newEntry),
+      });
+
+      if (response.ok) {
+        mutate();
+      }
+
+      if (!response.ok) {
+        console.log("Adding item failed");
```

Comment:
```suggestion
        console.error("Adding item failed");
```
Using console.error here would be more appropriate for logging errors, or?